_includes/postbox.html

```diff
@@ -8,11 +8,10 @@
                 {% if post.image %}
 
                     {% if site.lazyimages == "enabled" %}
-                        <img class="img-fluid lazyimg" src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAMAAAACCAQAAAA3fa6RAAAADklEQVR42mNkAANGCAUAACMAA2w/AMgAAAAASUVORK5CYII=" data-src="{% if post.image contains "://" %}{{ post.image }}{% else %}{{ site.baseurl }}/{{ post.image }}{% endif %}" alt="{{ post.title }}">
+                        <img class="img-fluid lazyimg" src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAMAAAACCAQAAAA3fa6RAAAADklEQVR42mNkAANGCAUAACMAA2w/AMgAAAAASUVORK5CYII=" data-src="{% if post.image-location contains "://" %}{{ post.image-location }}thumbnail_{{ post.image }}{% else %}{{ site.baseurl }}/{{ post.image-location }}thumbnail_{{ post.image }} }}thumbnail_{% endif %}" alt="{{ post.title }}">
                     {% else %}
-                        <img class="img-fluid" src="{% if post.image contains "://" %}{{ post.image }}{% else %}{{ site.baseurl }}/{{ post.image }}{% endif %}" alt="{{ post.title }}">
+                        <img class="img-fluid" src="{% if post.image contains "://" %}{{ post.image-location}}thumbnail_{{post.image }}{% else %}{{ site.baseurl }}/{{ post.image-location }}thumbnail_{{post.image }}{% endif %}" alt="{{ post.title }}">
```

Comment:
Instead of using `post.image-location`, put the image-location variable in the `_config.yml` file and use `site.post-location` instead. This is similar to pulling the `site.baseurl` variable.